client/src/common/permissions.ts

```diff
@@ -0,0 +1,21 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 (eventually AGPL-3.0) 2016-present Scille SAS
+
+import { useRoute } from 'vue-router';
+
+export function isAdmin(): boolean {
+  const currentRoute = useRoute();
+  const deviceId = currentRoute.params.deviceId;
+  if (deviceId) {
+    return true;
+  }
+  return false;
+}
+
+export function isOutsider(): boolean {
+  const currentRoute = useRoute();
+  const deviceId = currentRoute.params.deviceId;
+  if (deviceId) {
+    return false;
+  }
+  return false;
```

Comment:
This function always return `false`, is it the expected behavior?

Reply:
Yeah it's all mock for now and I don't have a good way to simulate this (random would screw up the tests, so it just returns false for now).